src/pages/download/_downloadPythonPackage.js

```diff
@@ -31,14 +35,15 @@ function getJSZipDateWithOffset() {
 
   return dateWithOffset;
 }
-async function generateZIP(deviceId) {
+
+export async function generateZIP(deviceId) {
   var zip = new JSZip();
   var count = 0;
   const zipFilename = !!deviceId ? `${deviceId}-mockup.zip` : "mockup.zip";
   var images = new Map();
-  dataurlkey = await allStorage();
+  var dataurlkey = await allStorage();
   dataurlkey.forEach(function (value, key) {
-    file = dataURLtoFile(value, key.substring(3, key.length) + ".png");
+    var file = dataURLtoFile(value, key.substring(3, key.length) + ".png");
```

Comment:
const/let does not work?

Reply:
yes, but other places in this file all using `var` :sweat_smile:, I can update them all at once when changing `.js` to .ts`